### DISCUSSION TITLE
Option to share custom performance profiles

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#609] [Ideas]
Hi,

First of all, thank you so much for creating such a helpful program.

The program works perfectly and has finally saved me from the scary Armory Crate.

I'm excited for the next update, "custom performance profiles", and it would be nice to be able to share custom performance profiles between users through files or something similar.

I believe most users want to change their performance settings, but don't know where to begin.

It would be great if profiles could be shared on sites like Reddit using files, allowing more users to enjoy this feature.

**Comment by seerge** [comment]
Hello, but you can already share all your settings by just taking config.json from ``%appdata%/ghelper``

**Reply by MrxSiN** [reply]
Probably what he mean was, just a portion of the config. Like a fan curve profile.
